pkg/generic/jsonthrift_codec.go

```diff
@@ -37,15 +39,32 @@ var (
 type JSONRequest = string
 
 type jsonThriftCodec struct {
-	svcDsc           atomic.Value // *idl
-	provider         DescriptorProvider
-	codec            remote.PayloadCodec
-	binaryWithBase64 bool
+	svcDsc                          atomic.Value // *idl
+	provider                        DescriptorProvider
+	codec                           remote.PayloadCodec
+	binaryWithBase64                bool
+	dynamicgoConvOpts               conv.Options // conversion options for dynamicgo
+	dynamicgoConvOptsWithThriftBase conv.Options // conversion options for dynamicgo with EnableThriftBase turned on
```

Comment:
There is no need to repeat the definition, but for multi t2j.BinaryConv singleton

Reply:
fixed